dash/dash.py

```diff
@@ -1228,6 +1240,28 @@ def callback(_triggers, user_store_data, user_callback_args):
                     # to pending key (hash of data requested by client)
                     user_store_data["current_key"] = pending_key
 
+                    if isinstance(result, dict) and result.get("long_callback_error"):
+                        error = result.get("long_callback_error")
+                        print(
+                            result["long_callback_error"]["tb"],
+                            file=sys.stderr,
+                        )
+                        return dict(
+                            error=f"An error occurred inside a long callback: {error['msg']}\n"
+                            + error["tb"],
+                            user_callback_output=NoUpdate(),
+                            in_progress=[val for (_, _, val) in running],
+                            interval_disabled=pending_job is None,
+                            progress=clear_progress,
+                            user_store_data=user_store_data,
+                        )
+
+                    if (
+                        isinstance(result, dict)
+                        and result.get("no_update") == "no_update"
```

Comment:
I guess having `no_update` create a value that's invalid as JSON would be tricky, particularly in the multi-output case but let's make the no_update serialization a little harder to encounter accidentally. I think we should (a) use the `_dash_` prefix we use for other internal-use values, ie `{"_dash_no_update": "_dash_no_update"}` , and (b) test for an exact match, not just this one key.

And re: multi-outputs, does it work to use `no_update` as one of several outputs to a long-callback? Can we include a test of this case?